### PR TITLE
chore: switch zio http implementation from d11 to dev.zio

### PR DESCRIPTION
**Pull request by BalduinLandolt**
### Task Description/Number

Issue Number: DEV-

### Basic Requirements

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

### PR Type

What kind of change does this PR introduce?

- [ ] Bugfix: represents bug fixes
- [ ] Refactor: represents production code refactoring
- [ ] Feature: represents a new feature
- [ ] Documentation: documentation changes (no production code change)
- [x] Chore: maintenance tasks (no production code change)
- [ ] Style: styles updates (no production code change)
- [ ] Test: all about tests: adding, refactoring tests (no production code change)
- [ ] Other... Please describe:

### Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### Does this PR change client-test-data?

- [ ] Yes (don't forget to update the JS-LIB team about the change)
- [ ] No

### CODE REVIEW

**Review by irinaschubert — APPROVED** [review]
I don't understand everything you did in the instrumentation server part, but apart from that it looks good to me. Just curious: why didn't you apply the changes for V3?

File: project/Dependencies.scala [hunk 1]
```diff
@@ -180,7 +182,7 @@ object Dependencies {
   val dspApiMainLibraryDependencies = Seq(
     zio,
     zioMacros,
-    zioHttp,
+    zioHttpOld, // TODO: switch
```
Comment (mpro7) [inline project/Dependencies.scala:185]: All 3 repetition of `// TODO: switch` could be removed.

**Review by mpro7 — APPROVED** [review]
Apart of some not removed comments looks good.

File: project/Dependencies.scala (hunk 1 above)
Comment (BalduinLandolt) [reply project/Dependencies.scala]: done

**Pull request merged**